velox/common/encode/Base64.h

```diff
@@ -41,119 +40,97 @@ class Base64 {
   /// within the encoding base.
   using ReverseIndex = std::array<uint8_t, kReverseIndexSize>;
 
-  /// Padding character used in encoding.
-  static const char kPadding = '=';
-
-  /// Encodes the specified number of characters from the 'data'.
-  static std::string encode(const char* data, size_t len);
+  /// Encodes the specified number of characters from the 'input'.
+  static std::string encode(std::string_view input, size_t len);
```

Comment:
This API should remain `static std::string encode(const char* data, size_t len);`